quinn-proto/src/packet.rs

```diff
@@ -770,6 +767,32 @@ impl PacketNumber {
     }
 }
 
+/// An [`ConnectionIdParser`] implementation that assumes the connection ID is of fixed length
```

Comment:
Nits: should be "A [`ConnectionIdParser`] implementation"

Reply:
Good call, fixed.